compute/mailjet/src/main/java/com/example/compute/mailjet/MailjetSender.java

```diff
@@ -34,23 +35,27 @@
   public static void main(String[] args) throws MailjetException, MailjetSocketTimeoutException {
     final String mailjetApiKey = "YOUR-MAILJET-API-KEY";
     final String mailjetSecretKey = "YOUR-MAILJET-SECRET-KEY";
-    MailjetClient client = new MailjetClient(mailjetApiKey, mailjetSecretKey);
+    MailjetClient client = new MailjetClient(mailjetApiKey, mailjetSecretKey, new ClientOptions("v3.1"));
 
     MailjetSender sender = new MailjetSender();
     sender.sendMailjet(args[0], args[1], client);
   }
 
   public MailjetResponse sendMailjet(String recipient, String sender, MailjetClient client)
   throws MailjetException, MailjetSocketTimeoutException {
-    MailjetRequest email = new MailjetRequest(Email.resource)
-        .property(Email.FROMEMAIL, sender)
-        .property(Email.FROMNAME, "pandora")
-        .property(Email.SUBJECT, "Your email flight plan!")
-        .property(Email.TEXTPART,
-            "Dear passenger, welcome to Mailjet! May the delivery force be with you!")
-        .property(Email.HTMLPART,
-            "<h3>Dear passenger, welcome to Mailjet!</h3><br/>May the delivery force be with you!")
-        .property(Email.RECIPIENTS, new JSONArray().put(new JSONObject().put("Email", recipient)));
+    MailjetRequest email = new MailjetRequest(Emailv31.resource)
```

Comment:
We don't use tabs, but do use two spaces, not 4.  If you could fix that, then I'll LGTM